packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -38,14 +38,21 @@
 // postcss-bem-linter: define text-field
 
 .mdc-text-field {
+  @include mdc-text-field-bottom-line-color($mdc-text-field-underline-on-light-idle);
+  @include mdc-text-field-bottom-line-hover-color($mdc-text-field-underline-on-light);
+  @include mdc-text-field-bottom-line-focused-color(primary);
+  @include mdc-text-field-label-ink-color($mdc-text-field-light-label);
+  @include mdc-text-field-input-ink-color(text-primary-on-light);
+  @include mdc-text-field-input-placeholder-ink-color(text-hint-on-light);
```

Comment:
combine the label-color, input-placeholder-ink-color (one is CSS only and one is JS)